components/ingest-service/backend/elastic/mappings/converge.go

```diff
@@ -244,6 +244,9 @@ var ConvergeHistory = Mapping{
 							},
 							"public_ipv4": {
 								"type": "ip"
+							},
```

Comment:
Should we make this a top level field? We could share the id with the tenant_id from azure and call it cloud_id or something for easier querying.